core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -1,6 +1,7 @@
-import { Component, AfterViewInit } from '@angular/core';
+import { Component, AfterViewInit} from '@angular/core';
 import { Observable } from 'rxjs/Observable';
 import '../../../common/rxjs-operators';
+import 'rxjs/add/observable/fromEvent';
```

Comment:
This line:
`import 'rxjs/add/observable/fromEvent';`
should be put into `app/common/rxjs-operators.ts` file, where we manage all the rxjs imports